queries/thrift/highlights.scm

```diff
@@ -9,96 +10,118 @@
   "cpp_include"
 ] @include
 
-(include_path) @string
-(package_path) @string
+; Function
 
-; Builtins
+(function_definition
+  (identifier) @function)
 
-(primitive) @type.builtin
+; Fields
 
-[
-  "list"
-  "map"
-  "set"
-  "sink"
-  "stream"
-  "void"
-] @type.builtin
+(field (identifier) @field)
 
-; Function
+; Parameters
 
-(function_identifier) @function
+(function_definition
+  (parameters
+    (parameter (identifier) @parameter)))
 
-; Fields
+(throws
+  (parameters
+    (parameter (identifier) @parameter.exception)))
```

Comment:
Why not just `@parameter`?

Reply:
mmm, in this case I'd find it useful for anyone using this to easily distinguish exception parameters, here's an example

```thrift
  map<binary,list<ColumnOrSuperColumn>> multiget_slice(1:required list<binary> keys,
                                                       2:required ColumnParent column_parent,
                                                       3:required SlicePredicate predicate,
                                                       4:required ConsistencyLevel consistency_level=ConsistencyLevel.ONE)
                                        throws (1:InvalidRequestException ire, 2:UnavailableException ue, 3:TimedOutException te),
```

the parameters after throws are exceptions

Reply:
I think `InvalidRequestException` should be `@exception` and `ire` should be `@parameter` and so on.

Reply:
> I think InvalidRequestException should be @exception

It's a type though, no? I guess it'd be nice to distinguish from regular types though

Reply:
e.g. in Python

![image](https://user-images.githubusercontent.com/29718261/224649943-aa44e90d-f122-4f19-98e4-85e9cdf9b24c.png)

Exceptions are still types.. :shrug: